docs/Installation.md

```diff
@@ -109,50 +113,47 @@ sudo systemctl reload apache2
 ```
 
 ### 4. FreeBSD
-For all commands below, acquire privileges, i.e. become root:
+
+* For all commands below become root:
 
 ``su -l``
 
-* Install Apache ([source](https://www.digitalocean.com/community/tutorials/how-to-install-an-apache-mysql-and-php-famp-stack-on-freebsd-10-1)):
+* Install Apache (see [tutorial on Apache on FreeBSD]) and its dependencies:
 
 ``pkg install apache24``
 
 Enter ``y`` at the confirmation prompt.
 
-This installs Apache and its dependencies.
-
-To enable Apache as a service, add ``apache24_enable="YES"`` to the ``/etc/rc.conf`` file. 
-We will use this sysrc command to do just that:
+* Enable Apache as a service:
 
 ``sysrc apache24_enable=yes``
  
-Now start Apache:
- 
-``service apache24 start``
+* Enable three apache modules in Apache configuration file:
 
-* Install Wget :
+``perl -pi -e 's/^#(LoadModule (proxy_module|proxy_http_module|rewrite_module) libexec)/$1/' /usr/local/etc/apache24/httpd.conf``
 
-```sh
-pkg install wget
-```
+* Start Apache:
+ 
+``service apache24 start``
 
 * Install Zonemaster Web GUI
 ```sh
-wget https://github.com/zonemaster/zonemaster-gui/releases/download/v3.0.1/zonemaster_web_gui.zip -O temp.zip
-mkdir -p  /var/www/html/zonemaster-web-gui
+fetch https://github.com/zonemaster/zonemaster-gui/releases/download/v3.0.1/zonemaster_web_gui.zip
```

Comment:
Should be 3.0.2

Reply:
That will be fixed at release time. We should probably have 3.0.3 since we have added things after.